### ISSUE TITLE
KeyError on execution

### ISSUE THREAD

**Issue by scossu** [Ashark/davinci-resolve-checker#26]
I am unable to run the checker on my system. 

OS: ArchLinux
GPU: Intel
Graphics server: Wayland (sway)
DaVinci Resolve 17.4.3 (from AUR)
Checker version: 2.4.1

```
$ ./davinci-resolve-checker.py 
Using locale en_US
DaVinci Resolve checker 2.4.1
Installed DaVinci Resolve package: davinci-resolve 17.4.3-1
Traceback (most recent call last):
  File "/home/stefano/code/davinci-resolve-checker/./davinci-resolve-checker.py", line 90, in <module>
    print(local_str["chassis"], local_str[chassis_type])
  File "/home/stefano/code/davinci-resolve-checker/local_strings.py", line 22, in __getitem__
    return self.data[key]
KeyError: 'Space-saving'
```

**Comment by Ashark** [comment]
You probably try to run the script alone, without languages files. They are required.

**Comment by scossu** [comment]
Thanks for the quick rely. Can you expand on that? The README has an example for an Nvidia card but I have an intel. Thanks.

**Comment by Ashark** [comment]
Have you cloned the whole repo or you just downloaded the single checker py file?

**Comment by scossu** [comment]
I cloned the repo.

**Comment by Ashark** [comment]
What is the output of `cat /sys/class/dmi/id/chassis_type`?

**Comment by scossu** [comment]
```
$ cat /sys/class/dmi/id/chassis_type
15
```

**Comment by Ashark** [comment]
https://github.com/Ashark/davinci-resolve-checker/blob/206e59942e325753fc6acc4b0a924fd5cb840805/davinci-resolve-checker.py#L52

I see it is something called "Space-saving".  I only made it for laptops and desktops currently. That is why the error occurred.
What is your hardware? Maybe there is a model number?

**Comment by scossu** [comment]
It's a Dell Optiplex desktop. What kind of hardware specs are you looking for precisely?

**Comment by Ashark** [comment]
Fixed in 2.5.0

**Comment by scossu** [comment]
Yes, I got past that step! Thanks a lot.
